sql/src/main/java/io/crate/action/sql/SQLOperations.java

```diff
@@ -136,7 +133,7 @@ public Session createSession(@Nullable String defaultSchema) {
         private Throwable throwable = null;
         private List<List<Object>> bulkParams = new ArrayList<>();
         private List<ResultReceiver> resultReceivers = new ArrayList<>();
-        private Map<String, String> settings;
+        private Map<String, String> settings = new HashMap<>();
```

Comment:
The idea behind putAll is that a SET [SESSION] param=value can happen at any time within the session so we should update the params.